### ISSUE TITLE
sentry-cli to upload to external symbol store

### ISSUE THREAD

**Issue by kristjanvalur** [getsentry/symbolicator#570]
So, Yesterday I started to work with symbol files in my project and _immediately_ hit the 2GB dif file barrier.  It was suggested to me via some of the issues here that the only workaround was to set up an external symbol store.
To even access this feature, I enabled the business plan for my org, but I was dismayed to find that the only thin that it did was to enable a read-only access to some external bucket.
No convenient way to actually add symbols.

sentry-cli is very convenient.  You point it at a folder and it trawls it for appropriate binaries, queries the upstream if they exist already, and only uploads those that matter.

Using an external symbol repo, to bypass sentry storage limits, removes this convenience completely.
Would it

-  Would it be hard to simply give sentry read/write access to an external bucket, like s3?  Do the file offloading automatically?
- Alternatively, allow the sentry-cli to talk directly to an external store, such as s3 or http, and upload directly, probably using the "unified" directory layout?

Basically, I just want a way to bypass the 2GB limit in a convenient way.  Also, being able to periodically prune my external store is convenient, but I can write my own tool to do that.

**Comment by dristic** [comment]
I was able to get it to work. The prefix appears to be simply a folder prefix and sentry does not take into account the prefix when searching for symbols. I used "linux" and then set my root folder to linux/ in Sentry and it appears to be working. I also set the bundle id to the sentry native SDK version but I'm not sure if that made any difference either.
